app/errors.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"errors"
+	"log"
+	"net/http"
+)
+
+var (
+	errNotFound = errors.New("not found")
+	// TODO: Add errors corresponding to http status codes
+	// that should be returned by the server.
+)
+
+func errHandler(w http.ResponseWriter, err error) {
+	if errors.Is(err, errNotFound) {
+		log.Printf("%v", err)
```

Comment:
log is intended?

Reply:
Yeah this is for internal debugging (server-side).